pkg/kubectl/cmd/set/env/env_resolve.go

```diff
@@ -23,8 +23,8 @@ import (
 	metav1 "k8s.io/apimachinery/pkg/apis/meta/v1"
 	"k8s.io/apimachinery/pkg/runtime"
 	"k8s.io/client-go/kubernetes"
-	"k8s.io/kubernetes/pkg/api/v1/resource"
-	"k8s.io/kubernetes/pkg/fieldpath"
+	"k8s.io/kubernetes/pkg/kubectl/util/fieldpath"
+	"k8s.io/kubernetes/pkg/kubectl/util/resource"
```

Comment:
Similarly copy just the contents of `ExtractContainerResourceValue` into `getResourceFieldRef`.